src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write code here
+const inputs = document.querySelectorAll('.field-text');
+
+for (const input of inputs) {
+  const label = document.createElement('label');
+
+  label.className = ['field-label'];
+  label.htmlFor = input.id;
+  label.textContent = input.name;
+
+  input.parentNode.append(label);
+  input.placeholder = editingPlaceholder(input.name);
+}
+
+function editingPlaceholder(string) {
+  const letters = [];
+
+  for (const symbol of string.slice(1)) {
+    if (symbol === symbol.toUpperCase()) {
+      letters.push(' ', symbol.toLowerCase());
+    } else {
+      letters.push(symbol);
+    }
+  }
+
+  return string[0].toUpperCase() + letters.join('');
+}
```

Comment:
```suggestion
  let placeholder = input.name;

  input.placeholder = placeholder[0].toUpperCase() + placeholder.slice(1);
}
```
С плейсхолдером сильно заморочился) А в остальном всё гуд.